.github/release.yml

```diff
@@ -10,6 +10,24 @@ changelog:
         labels:
           - not-testable
           - maintenance
+          - web
+          - native
+    - title: [web] Features, fixes, enhancements 🎉
+      labels:
+        - web
+      exclude:
+        labels:
+          - not-testable
+          - maintenance
+          - native
+    - title: [native] Features, fixes, enhancements 🎉
+      labels:
+        - native
```

Comment:
Do you think we should also add specific android/ios groups? Or should we handle those by using [android]/[ios] in the PR title?